composer.json

```diff
@@ -21,6 +21,11 @@
     "config": {
         "sort-order": true
     },
+    "autoload": {
+        "psr-4": {
+            "VariableAnalysis\\": "VariableAnalysis/"
+        },
```

Comment:
This trailing comma is not valid JSON (I wish it was!) and causes composer to die (try `composer test`).